packml_ros/src/packml_ros.cpp

```diff
@@ -362,4 +353,111 @@ bool PackmlRos::loadStats(packml_msgs::LoadStats::Request &req, packml_msgs::Loa
 
   return true;
 }
+
+bool PackmlRos::commandGuard(const int& command_int, bool& command_valid, bool& command_rtn)
+{
+  command_valid = true;
+  command_rtn = false;
```

Comment:
Unless I'm reading this method wrong, I think you could get by with just one of these variables: set it true initially, then false by either the the calls to sm_ or the default case, and then return it. Not a big deal though.

Reply:
The distinction here is in isolating what went wrong in commandGuard. `command_valid` specifies whether the command is in the standard (e.g. `foo` is not a valid command in the PackML standard) while `command_rtn` indicates whether the command can be accommodated in the current state (e.g. Start is a valid command, but cannot be issued from the ABORTED state).

Reply:
Sure, that makes sense too.